test/D04/TC01.cs

```diff
@@ -0,0 +1,44 @@
+using aoc.D04;
+using Xunit;
+
+namespace aoc.test.D04
+{
+  public class TC01
+  {
+    [Fact]
+    public void InitialI()
+    {
+      int expected = 2;
+      var pairs = Parser.Parse(@"./D04/initial.txt");
+      var actual = PairAnalyzer.GetFullyCotainedPairsCount(pairs);
+      Assert.Equal(expected, actual);
+    }
+
+    [Fact]
```

Comment:
you could use `[Theory]` and `[InlineData]`

Reply:
does not help much, inputs are different called methods are different for partI n II